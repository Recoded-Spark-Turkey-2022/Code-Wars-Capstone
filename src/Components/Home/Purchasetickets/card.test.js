
import React from "react";
import renderer from "react-test-renderer";
import Card from "./Navbar";
import { MemoryRouter } from "react-router-dom";

it("renders correctly", () => {
    const test = renderer
        .create(
            <MemoryRouter>
                <Card/>
            </MemoryRouter>
        )
        .toJSON();
    expect(test).toMatchSnapshot();
});