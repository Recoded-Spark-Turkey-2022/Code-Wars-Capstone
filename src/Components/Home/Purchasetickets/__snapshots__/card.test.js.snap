// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card renders correctly 1`] = `
<div
  className="flex flex-col shadow-lg   px-10 py-16  text-center rounded-lg w-3/12  "
>
  <h1
    className=" lg:text-5xl md:text-2xl	sm:text-sm  lg:pt-8 md:pt:6 sm:pt:4 "
  >
     
  </h1>
  <p
    className="lg:text-2xl md:text-1xl	sm:text-sm text-stone-700 lg:pt-8 md:pt:6 sm:pt:4   lg:pb-6 md:pb:4 sm:pb:2 "
  >
     
  </p>
  <button
    className="lg:text-2xl md:text-1xl sm:text-sm rounded-md box-border p-2 transition-all duration-250 bg-cyan-400 hover:bg-cyan-500 hover:text-white"
    onClick={[Function]}
    type="submit"
  >
     Purchase 
  </button>
</div>
`;
